src/index.js

```diff
@@ -146,7 +148,7 @@ export default class SuperImage extends React.Component {
 
     return (
       <div
-        role="img"
+        role={this.props.role || 'img'}
         aria-label={this.props.alt}
```

Comment:
https://github.com/openfresh/super-image/pull/27/files#r160867631

> Aria roles which MAY be used on img element are only two roles, presentation and none. And only case that alt="".

So, it can use this.props.role when this.props.alt is undefined. and `img` element is the same case.

Reply:
@masuP9 That makes sense.

For example, is the specify as follows correct? (I am worried that there is a misunderstanding ...)

```diff
return (
  <div
-   role={this.props.role || 'img'}
+   role={this.props.alt ? 'img' : this.props.role}
    aria-label={this.props.alt}
    className={this.props.className}
    style={style}
  />
);
```

Reply:
@tsuyoshiwada That's it! Then add 'img' for when this.props.role is undefined.

```diff
return (
  <div
-   role={this.props.role || 'img'}
+   role={this.props.alt ? 'img' : this.props.role || 'img'}
    aria-label={this.props.alt}
    className={this.props.className}
    style={style}
  />
);
```

How is it?

Reply:
@masuP9 It looks good !
I will also fix `<img />` together :+1: